app/javascript/article-form/components/__tests__/ArticleCoverImage.test.jsx

```diff
@@ -95,7 +95,7 @@ describe('<ArticleCoverImage />', () => {
       expect(queryByText(/uploading.../i)).not.toBeInTheDocument();
       expect(queryByLabelText('Add a cover image')).not.toBeInTheDocument();
       expect(getByRole('img', { name: 'Post cover' })).toBeInTheDocument();
-      expect(getByLabelText('Change')).toBeInTheDocument();
+      expect(getByLabelText('Change', { exact: false })).toBeInTheDocument();
```

Comment:
Not sure if `{ exact: false }` is the best use here? Maybe using regex would be better? For example:
```js
getByLabelText(/Change.*/i)
```